### ISSUE TITLE
Idea/Funcionalidad: cruce de lemas y acepciones

### ISSUE THREAD

**Issue by mariavarelar** [jgsogo/neutron#10]
No estoy segura de si esto es un error o una trampa. En todo caso, es una muestra de lo delicado que es cruzar lemas y acepciones para cazar informantes despistados. "Bebé" podría de hecho entenderse en ciertos contextos como "que empieza a aparecer" (en el sentido de "estar en pañales"), pero no es una definición de nuestro diccionario.
![image](https://cloud.githubusercontent.com/assets/16288726/16199623/9c85e470-370a-11e6-9b1e-719592eb987d.png)
Voto por descartar este sistema y utilizar trampas supervisadas como "jordongo".
Por otra parte, no parece razonable que una trampa tenga continuidad en la modalidad Word alternate.
![image](https://cloud.githubusercontent.com/assets/16288726/16199887/97336a96-370b-11e6-97e2-63fe4410759e.png)

**Comment by jgsogo** [comment]
**No es una trampa, es una validación cruzada.**

Comento por qué se está dando esta situación y así podemos valorarla mejor.

1. Existe un informante, jgsogo[Bolivia] al que se le mostró la palabra "X" con el significado "Que empieza a aparecer".
2. jgsogo[Bolivia] indicó que no conocía esa palabra y eventualmente se le presentó en "WordAlternate" para que sugiriera cuál es la palabra que él utiliza para "Que empieza a aparecer".
3. jgsogo[Bolivia] escribe "Bebé" para ese significado.

A continuacióin apareces tú, informante de España y, por lo tanto, se te van a mostrar las palabras aportadas por informantes que no sean españoles; en este momento sólo existe la palabra bebé (ver #11 ) y te aparece.

Hay que tomar una de estas decisiones:
* Sólo mostrar palabras de informantes cualificados (está claro que en este caso jgsogo[Bolivia] no lo es). Cualificar a un informante sería una decisión manual (hasta que se nos ocurra cómo hacer esto automático) y siempre podría haber incongruencias de este tipo.
* Dejarlo abierto y no preocuparse por ello.

**Comment by mariavarelar** [comment]
Claro, no había entendido el origen de esta entrada. Está clarísimo ahora.
¿No podríamos cualificar a los informantes en función de su nivel educativo, el grado de coincidencia con otros informantes de su país y el número de palabras sobra las que han informado? No eliminaremos el 100% de las tonterías, pero sí una cantidad que puede merecer la pena, ¿no?

**Comment by jgsogo** [comment]
Mi idea es utilizar, como dices, _el grado de coincidencia con otros informantes de su país y el número de palabras sobra las que han informado_ (no me fío del nivel de estudios que ellos mismos han puesto y que se pueden haber inventado). El problema es el del huevo y la gallina:

Para puntuar a un informante necesito medir cuánto coincide con otros informantes (típicamente con los "buenos"), pero para tener informantes identificados como buenos necesitos puntuarlos, y para puntuar necesito comparar con los buenos y para identificar a los buenos necesito puntuar.... se me ocurren cosas, pero podrían no funcionar y sólo se pueden aplicar en un contexto de "grandes números".

**Comment by mariavarelar** [comment]
Creo que tengo una idea para resolver la situación. Hagamos algo semiautomático:

**Escenario A:**

- Un informante aporta una palabra alternativa para una acepción.
- Esta alternativa nunca había sido sugerida para esta acepción.
- El supervisor (nuevo rol) recibe el aviso de que hay una nueva alternativa en el sistema. La revisa, corrige la ortografía si es necesario, comprueba que no ha sido un lapsus y la aprueba.
- La alternativa se une al flujo de palabras.

**Escenario B:**

- Un informante aporta una alternativa para una acepción.
- Esta alternativa ya había sido sugerida antes y aprobada por el supervisor, por lo que se une de forma automática al flujo de palabras.

¿Puede ser un equilibrio razonable?

**Comment by jgsogo** [comment]
La solución del **supervisor** es factible, pero yo no la llamaría automática, ni semi, es totalmente manual; pero está claro que puede ser la única opción para empezar. Una vez que una pareja palabra-acepción está validada por el supervisor y se incorpora al flujo, ya no hay que hacer nada con el resto de informantes que la sugieran, ya está añadida ;D

 * Etiquetaríamos las palabras supervisadas dándoles una credibilidad del 100% ;D
 * De momento los supervisores serán los que tengan acceso a la interfaz de administración.
